guidelines/index.html

```diff
@@ -2683,7 +2683,7 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 <dd class="new"><p class="change">[New]</p>
    					
    <p>solidly bounds or surrounds</p>
-   				
+   <div class="note"><div role="heading" class="note-title marker" aria-level="5"><span>訳注</span></div><p>この用語は、WCAG 2.2 の 2023 年 10 月 5 日付けの勧告においては、削除されるべきであるにもかかわらず残っているものであり、いずれ正式に削除される予定である。</p></div>
```

Comment:
他の注記はレベル3で提供されているので、それに合わせるのがよいかと。
<https://raw.githack.com/waic/wcag22/main/guidelines/index.html#h-note-56>

状況としてはシンプルに
「この文書内でどこからも参照されていない」ことを示せばよいのかなと思いました。以下はどうでしょう…？


```suggestion
   <div class="note"><div role="heading" class="note-title marker" aria-level="5"><span>訳注</span></div><p>この用語は、この文書内でどこからも参照されていないものであり、正式に削除される予定である。</p></div>
```

Reply:
aria-level の値、すみません、見落としていました...！
文面の修正案も、ありがとうございます。おっしゃるように、このくらい簡潔で十分かと思います。